src/components/InformationsTable.tsx

```diff
@@ -1,60 +1,71 @@
+/* eslint-disable react/no-array-index-key */
 import clsx from 'clsx';
-import { useTranslations } from 'next-intl';
 import type { HTMLProps, ReactNode } from 'react';
 
-import { Table, TableBody, TableCaption, TableCell, TableCellHeader, TableRow } from '@/ui/table/Table';
+import { Table, TableBody, TableCaption, TableCell, TableCellHeader, TableHead, TableRow } from '@/ui/table/Table';
 
 export type InformationsTableHeaderWidth = 'xs' | 'sm' | 'md';
 
 export type InformationsTableRow<T extends object> = {
   header: ReactNode | string;
-  accessorKey: keyof T;
+  accessorKey?: keyof T;
   renderCell?: (row: T) => ReactNode;
   cellProps?: HTMLProps<HTMLTableCellElement>;
 };
 
 type InformationsTableProps<T extends object> = {
   rows: InformationsTableRow<T>[];
   data: T | T[];
+  headers?: HTMLProps<HTMLTableCellElement>[];
   caption?: string;
   headerCellWidth?: InformationsTableHeaderWidth;
   reducedFont?: true;
+  className?: string;
 };
 
-const InformationsTable = <T extends object>({ rows, data, caption, headerCellWidth = 'md', reducedFont }: InformationsTableProps<T>) => {
-  const t = useTranslations('Tables.Objects');
-
-  const renderTableCell = (accessorKey: keyof T, data: T, renderCell?: (row: T) => ReactNode, reducedFont?: true) => (
+const InformationsTable = <T extends object>({ rows, data, headers, caption, headerCellWidth = 'md', reducedFont, className }: InformationsTableProps<T>) => {
+  const renderTableCell = ({
+    data,
+    accessorKey,
+    renderCell,
+    reducedFont,
+  }: {
+    data: T;
+    accessorKey?: keyof T;
+    renderCell?: (row: T) => ReactNode;
+    reducedFont?: true;
+  }) => (
     <TableCell className={`${reducedFont ? 'text-base' : ''}`}>
-      {renderCell ? renderCell(data) : data[accessorKey] as ReactNode}
+      {renderCell ? renderCell(data) : accessorKey ? data[accessorKey] as ReactNode : '-'}
     </TableCell>
   );
 
   return (
-    <Table>
+    <Table className={className}>
       {caption && <TableCaption>{caption}</TableCaption>}
-      {Array.isArray(data) && (
-        <TableRow>
-          <TableCell className="w-80" />
-          <TableCellHeader>{t('primary')}</TableCellHeader>
-          <TableCellHeader>{t('secondary')}</TableCellHeader>
-        </TableRow>
+      {headers && (
+        <TableHead>
+          <TableRow>
+            {headers.map((header, index) => <TableCellHeader key={index} {...header} />)}
```

Comment:
We get "Each child in a list should have a unique "key" prop." 

This key is not unique for Tables which have more than 2 columns.
F.e. Object Data on Conjunction page